src/utils/api.js

```diff
@@ -0,0 +1,4 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export const request = url => fetch(`${BASE_URL}${url}`)
+  .then(response => response.json());
```

Comment:
```suggestion
  .then(response => response.json())
  .then(result => result.data);
```
Можно сразу тут data брать